script/cmake-ck-dev.sh

```diff
@@ -7,7 +7,7 @@ MY_PROJECT_SOURCE=$1
 
 cmake                                                                                             \
 -D CMAKE_PREFIX_PATH=/opt/rocm                                                                    \
--D CMAKE_CXX_COMPILER=/opt/rocm/bin/hipcc                                                         \
+-D CMAKE_CXX_COMPILER=/opt/rocm-5.7.1/bin/hipcc                                                         \
```

Comment:
Please not change this to `/opt/rocm-xxx`, since this script will be used by other developers, with different rocm version. If need a specific rocm version, or on your environment you don't have `/opt/rocm/` but only `opt/rocm-5.7.1`, you can have a soft link to `/opt/rocm/` by cmd e.g, `ln -s /opt/rocm-5.7.1 /opt/rocm`